perses/app/setup_relative_calculation.py

```diff
@@ -166,7 +199,7 @@ def run_setup(setup_options):
 
     else:
         eq_splitting = "V R O R V"
-        neq_splitting = "V R O H R V"
+        neq_splitting = "V R O R V"
```

Comment:
How come this has changed?

Reply:
You need the `H` if you want to accumulate work and actually switch the potential!

Reply:
we are not updating the hamiltonian in the integrator; it is being updated between single-step md moves manually